examples/dit/data/dataset.py

```diff
@@ -0,0 +1,282 @@
+import csv
+import logging
+import os
+
+import albumentations
+import cv2
+import numpy as np
+from PIL import Image
+
+import mindspore as ms
+
+logger = logging.getLogger()
+
+
+def create_video_transforms(h, w, num_frames, interpolation="bicubic", backend="al", use_safer_augment=True):
```

Comment:
not necessary

Reply:
I'm using `create_video_transforms` as image pixel transformation by setting `num_frames=1`. I will change it to another name to avoid confusion.